chapter2/tests/t19/terraform/_modules/nginx/main.tf

```diff
@@ -0,0 +1,31 @@
+module "container" {
+  source                       = "../container"
+  container_image              = var.container_image
+  container_image_keep_locally = var.container_image_keep_locally
+  container_name               = var.container_name != null ? var.container_name : ""
+  container_ports              = var.container_ports
+  client                       = var.client
+  project                      = var.project
+  environment                  = var.environment
+  container_volumes            = var.container_volumes
+}
+
+resource "null_resource" "index_page" {
+  provisioner "local-exec" {
+    command = <<EOF
+    mkdir ${path.cwd}/${var.environment} && 
+    cat > ${path.cwd}/${var.environment}/index.html  <<EOL
```

Comment:
EOL?

Reply:
> EOL?

@dmitry-mightydevops
It seems more correct now, done